src/EditorFeatures/Test2/Rename/InlineRenameTests.vb

```diff
@@ -2383,5 +2383,48 @@ class [|C|]
             End Using
         End Function
 
+        <WpfTheory>
+        <CombinatorialData, Trait(Traits.Feature, Traits.Features.Rename)>
+        Public Sub RenameRecordParameter(host As RenameTestHost)
+            Using result = RenameEngineResult.Create(_outputHelper,
+                    <Workspace>
+                        <Project Language="C#" CommonReferences="true">
+                            <Document>
+                                public record MyRecord(string [|$$MyProperty|]);
```

Comment:
can you have a test whre this is `MyRecord<T>`, this is `T MyProperty` and you're renaming at an instantiation point where yo uhave a `MyRecord<int>`?  If you do that and it works (which i give you 50/50 odds on) then i can sign off.  tnx.

Reply:
Added.